include/mbedtls/config.h

```diff
@@ -3756,6 +3756,14 @@
  */
 //#define MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED
 
+/**
+ * Internal define that removes the zeroization of the output when decrypting
+ * CCM and the tag check fails. This is for internal use only, and was added so
+ * that PSA multipart CCM could be implmented. This option will be removed at
+ * some point in the future when proper CCM multipart support is implemented.
+ * Use at own risk.
+ */
+//#define MBEDTLS_INTERNAL_CCM_NO_ZEROIZE_ON_TAG_FAIL
```

Comment:
Blocker: no.

I thought we'd decided not to implement CCM since it can't be done sensibly.

It doesn't make sense to have a configuration option “for internal use only”. An API function could be for internal use only, but a compile-time option cannot: the only way it can be used internally is to enable it.

This option only lets us pass PSA compliance tests (with CCM only, not GCM) if it's enabled. It doesn't even implement the whole API, just enough of it to pass part of compliance. This is not useful. We want to pass compliance with a normal build, and should not expend energy on having a special temporary mode just to pass compliance (but only partially) a few weeks earlier.

Please remove this option and all associated code.